src/test/java/api/support/fixtures/LoansFixture.java

```diff
@@ -104,6 +107,15 @@ public Response attemptToCreateLoan(
     return response;
   }
 
+  public IndividualResource declareItemLost(
+    DeclareItemLostRequestBuilder builder) {
+    JsonObject request = builder.create();
+
+    return new IndividualResource(from(post(request,
```

Comment:
Does this mean there is no need for the put method for rest assured?

Reply:
yes, it's been changed.